randovania/games/samus_returns/generator/bootstrap.py

```diff
@@ -64,6 +64,9 @@ def _get_enabled_misc_resources(
             if getattr(configuration, name):
                 enabled_resources.add(index)
 
+        if configuration.dock_rando.is_enabled() or False:
```

Comment:
Can't see the purpose of `or False`.

Reply:
Unless it was a fluke, when I didn't do this codecov/patch failed with 0% coverage somehow

Reply:
It is still the case and not a fluke. There is only one coverable line (ignoring the condition because it has no `else` part) for the changes and that line is not covered which makes it 0 % :D. We don't have a test for the generator of MSR with the door lock option activated. Hence the line cannot be covered by a test.
I would suggest to add tests when door lock is really added and ignore it for now.
But remove the nonsense `or False` part :p